test/e2e_node/conformance_test.go

```diff
@@ -62,7 +62,7 @@ var _ = Describe("Container Conformance Test", func() {
 				phase := api.PodPending
```

Comment:
If that's the intention, I feel like you should add a sleep to the container run command, and verify it enters the `Running` phase.